http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -665,6 +665,16 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
       context <- ctx.ask
       // should num workers be updated?
       targetNumWorkers <- traverseIfChanged(req.updatedNumberOfWorkers, dataprocConfig.numberOfWorkers)(Async[F].pure)
+      _ <- targetNumWorkers.traverse(_ =>
+        if (runtime.status != RuntimeStatus.Running)
+          F.raiseError[Unit](
+            new LeoException(
+              s"${context.traceId.asString} | Bad request. Number of workers can only be updated if the dataproc cluster is Running. Please start your runtime before updating the cluster",
```

Comment:
The last sentence in the error message sounds like it assumes the status is `Stopped` even though the predicate is any status but `Running`. I assume the code is correct, in which case we may want to add to the message what the status is and remove `Please start your runtime before updating the cluster`.

Reply:
I'll update the check it `Stopped`

Reply:
You mean changing it to 
```scala
if (runtime.status != RuntimeStatus.Stopped)
```
?

If so, what will happen when `runtime.status` is not `Running` or `Stopped`? My understanding was that we can update a cluster only if its status is `Running`, no?

Reply:
No, it's changed to `runtime.status == RuntimeStatus.Stopped`